From a.beregalov@gmail.com  Fri Jul 10 15:23:30 2009
From: Alexander Beregalov <a.beregalov@gmail.com>
Date: Fri,  3 Jul 2009 16:25:53 +0400
Subject: Staging: wlan-ng: remove dependency on WIRELESS_EXT version
To: gregkh@suse.de, linux-kernel@vger.kernel.org
Cc: Alexander Beregalov <a.beregalov@gmail.com>
Message-ID: <1246623953-23215-4-git-send-email-a.beregalov@gmail.com>


As the driver is in mainline now we can remove such dependency.
WIRELESS_EXT is 22 now and it is always defined.

Signed-off-by: Alexander Beregalov <a.beregalov@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/wlan-ng/prism2sta.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/staging/wlan-ng/prism2sta.c
+++ b/drivers/staging/wlan-ng/prism2sta.c
@@ -1444,9 +1444,7 @@ void prism2sta_processing_defer(struct w
 	}
 
 	wlandev->linkstatus = (hw->link_status == HFA384x_LINK_CONNECTED);
-#ifdef WIRELESS_EXT
 	p80211wext_event_associated(wlandev, wlandev->linkstatus);
-#endif
 
 failed:
 	return;
